Source/i_system.c

```diff
@@ -257,7 +257,7 @@ void I_Quit (void)
       G_CheckDemoStatus();
    M_SaveDefaults();
 
-#if defined(MY_SDL_VER)
+#if defined(MY_SDL_VER) && defined(_MSC_VER)

```

Comment:
I just had a look, and it seems that `getch()` is available in mingw-w64 at least. Don't know about the older mingw32 project, but I don't think it should be that much of a concern.